go/api/commands.go

```diff
@@ -132,7 +132,13 @@ type StringCommands interface {
 	// Sets multiple keys to multiple values in a single operation.
 	//
 	// Note:
-	//  When in cluster mode, the command may route to multiple nodes when keys in keyValueMap map to different hash slots.
+	//   In cluster mode, if keys in `keyValueMap` map to different hash slots,
+	//   the command will be split across these slots and executed separately for each.
+	//   This means the command is atomic only at the slot level. If one or more slot-specific
```

Comment:
Consider adding this to the wiki page.

Reply:
Done: 
https://github.com/valkey-io/valkey-glide/wiki/General-Concepts#multi-slot-command-handling